### DISCUSSION TITLE
recordings are stopping halfway through motion

### DISCUSSION THREAD

**Discussion by the1robbo** [blakeblackshear/frigate#3551] [General]

**Comment by NickM-27** [comment]
No need to remove the description if you figured it out. Better that you answer your own question as others will come along with a similar one most likely

**Reply by the1robbo** [reply]
I realised it was a stupid question and went to reword it but I lost the whole lot

**Reply by NickM-27** [reply]
Gotcha, if you look at the edited history you can get it back

**Comment by the1robbo** [comment]
Hi,

For some reason my recordings are stopping even when motion is still taking place.

Any setting that stops this from happening please?

Setup is:
ffmpeg:
input_args:
- -avoid_negative_ts
- make_zero
- -fflags
- nobuffer+genpts+discardcorrupt
- -flags
- low_delay
- -strict
- experimental
- -analyzeduration
- 1000M
- -probesize
- 1000M
- -rw_timeout
- "5000000"
inputs:
- path: http://x/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=x&password=x
roles:
- record
- rtmp
- detect
motion:
mask:
- "1280,0,1280,82,893,82,546,133,506,0,668,0"
objects:
track:
- person
- car
filters:
person:
threshold: 0.7
min_score: 0.7
snapshots:
enabled: True
timestamp: True
bounding_box: True
retain:
default: 30
record:
enabled: True
events:
retain:
default: 90

**Reply by NickM-27** [reply]
What do you mean by recordings stopping? Like when you watch them back the event stops early?

Right now you don't have 24/7 recordings enabled, only events recording. So motion occurring doesn't matter if the object is no longer in view of the camera.

**Reply by the1robbo** [reply]
In the events section it shows me entering and then moving the bins :) but before I have left the camera, the recording has finished.  

If I go to the media ->frigate -> recordings then select the month, day and time it's all there. 

Is the event just to show you a brief section of the recording?

**Reply by NickM-27** [reply]
Interesting, no the events will show you from when the object is visible to when it is not visible. Are there multiple events? It wouldn't make sense for the recording to be saved but not included with an event given the config you provided.

**Comment by the1robbo** [comment]
This should have been a single recording...

![image](https://user-images.githubusercontent.com/106261708/181514583-8f633d75-f24c-40bb-a431-3d777fa8960b.png)

**Reply by NickM-27** [reply]
If it loses the object then it will be split into separate recordings regardless if motion continues

**Comment by the1robbo** [comment]
gotcha.

**Reply by NickM-27** [reply]
If you find it happens often, you can try to slightly incrase `detect -> max_disappeared` to increase the time it takes to consider the object gone and start a new event. Increasing too much runs the risk of blending separate objects into the same event when they are different
